tests/test-vandv/src/main/java/mil/emp3/test/emp3vv/containers/dialogs/milstdunits/SymbolPropertiesDialog.java

```diff
@@ -394,6 +410,26 @@ public void onViewCreated(View view, final Bundle savedInstanceState) {
 
         super.onViewCreated(view, savedInstanceState);
 
+        // Modifier stuff
+        modifierMap.clear();
+        modifierSpinner = (Spinner) view.findViewById(R.id.modifier_type);
+        modifierValue = (EditText) view.findViewById(R.id.modifier_value);
+
+        modifierList = new ArrayList<>();
+        for(IGeoMilSymbol.Modifier modifier : IGeoMilSymbol.Modifier.values()) {
+            modifierList.add(modifier.toString());
+        }
+        modifierTypeAdapter = new ArrayAdapter<CharSequence>(this.getContext(), android.R.layout.simple_spinner_item, modifierList);
+        modifierSpinner.setAdapter(modifierTypeAdapter);
+        // Modifier Button
+        Button addModifierButton = (Button) view.findViewById(R.id.add_modifier_button);
+        addModifierButton.setOnClickListener(v -> {
+            modifierMap.put(IGeoMilSymbol.Modifier.valueOf(((String)modifierSpinner.getSelectedItem())), modifierValue.getText().toString());
+            Toast.makeText(this.getContext(), "Added modifier", Toast.LENGTH_SHORT).show();
+        });
+
+
```

Comment:
remove extra lines (there are 3 could go down to just 1)

Reply:
Fixed